kedro_kubeflow/config.py

```diff
@@ -195,6 +200,12 @@ def volume(self):
     def wait_for_completion(self):
         return bool(self._get_or_default("wait_for_completion", False))
 
+    @property
+    def expose_kedro_outputs_as_kfp_artifacts(self):
```

Comment:
`store_kedro_outputs_as_kfp_artifacts`?

Reply:
I'm not convinced, I kind of like the "expose" naming, as the plugin doesn't store anything in MLMD. Maybe @em-pe / @michalzelechowski-getindata have some strong opinion?

Reply:
OK, renamed, since I need to merge this PR :-)